package.json

```diff
@@ -31,6 +31,7 @@
         "@types/uuid": "^9.0.4",
         "axios": "^1.5.1",
         "cors": "^2.8.5",
+        "crypto-js": "^4.2.0",
```

Comment:
CryptoJS is [no longer supported](https://www.npmjs.com/package/crypto-js), use [node's Crypto](https://nodejs.org/api/crypto.html) instead.